install-manifests/azure-container-with-pg/azuredeploy.json

```diff
@@ -97,7 +97,7 @@
     "firewallRuleName": "allow-all-azure-firewall-rule",
     "containerGroupName": "[concat(parameters('name'), '-container-group')]",
     "containerName": "hasura-graphql-engine",
-    "containerImage": "hasura/graphql-engine:v1.1.1"
+    "containerImage": "hasura/graphql-engine:v1.2.0-beta.4"
```

Comment:
are we okay to put beta images in manifests?

Reply:
Yes, docs and everything else points to `stable` branch